docs/conversion_example_gallery.rst

```diff
@@ -0,0 +1,32 @@
+Conversion Gallery
```

Comment:
I'd put this file inside the conversion_examples_gallery directory. Then all of the reference can shorten, e.g. conversion_examples_gallery/recording/axona -> recording/axona

Reply:
I was thinking on having everything that is on `index.rst` (the top-level file) also in docs. The idea was the file structure would reflect the document structure.